src/fetch/payouts.py

```diff
@@ -344,10 +344,16 @@ def post_cip20_payouts(
     price_day = dune.period.end - timedelta(days=1)
     reward_token = TokenId.COW
 
+    quote_rewards_df = orderbook.get_quote_rewards(dune.start_block, dune.end_block)
+    batch_rewards_df = orderbook.get_solver_rewards(dune.start_block, dune.end_block)
+    merged_df = pandas.merge(
```

Comment:
If I understand correctly, this means that the two payouts (solver and quote) remain completely disjoint, right? I think this is the cleanest approach, and the only "bad" thing about this is that it would lead to doing twice as many COW transfers as we are doing now. But I think it's fine, since verification will remain straightforward, and in any case even now we are a bit wasteful in our transfers.

Reply:
That logic is not implemented and it is up to us to decide how we want to use the data. We could merge payments if we wanted. You seem to prefer separate payments?
The main advantage of merging other payments (slippage, performance rewards, consistency rewards) was fewer overdrafts and less transfers. Some disadvantages are e.g. that payments are not independent and an error in one of them makes the total payment wrong, or that the logic for computing payments becomes harder to check.

Reply:
Note that we will need to ensure we are doing a _full outer join_ here and filling null values with zeroes something like `pandas.fillna(0)`.

This would make up for scenarios where solvers are solving, but not quoting and vice versa. Does CIP-27 say anything about solvers to only give quotes and do not solve? Because they do not even need to be registered solvers to do this.

Reply:
Per CIP-27, all estimators need to be solvers, and the estimation rewards will be sent to the corresponding solver rewards addresses.

However, it can indeed happen that a solver solves nothing but successfully quotes a few orders.

In any case, I would strongly suggest to keep things completely separate so that debugging/verifying is easier.

Reply:
It appears this is what has been done, so I will resolve this issue.